articles/active-directory-b2c/b2clogin.md

```diff
@@ -17,6 +17,9 @@ ms.subservice: B2C
 
 When you set up an identity provider for sign-up and sign-in in your Azure Active Directory (Azure AD) B2C application, you need to specify a redirect URL. In the past, login.microsoftonline.com was used, now you should be using b2clogin.com.
 
+>[!NOTE]
```

Comment:
Thanks for the update, @juntakata. Recommend that instead of a NOTE (these are often skipped by readers) that we instead add another bullet to the **Benefits** list, simply saying:

* You can use JavaScript in your HTML page layouts for [user flows](javascript-samples.md) and [custom policies](user-flow-javascript-overview.md) (JavaScript support is currently in preview)

Raw version:
```Markdown
* You can use JavaScript in your HTML page layouts for [user flows](javascript-samples.md) and [custom policies](user-flow-javascript-overview.md) (JavaScript support is currently in preview)
```

For the HTML element removal, it should not be mentioned here, and instead in the UI customization article(s) such as:

* https://docs.microsoft.com/en-us/azure/active-directory-b2c/customize-ui-overview
* https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-ui-customization-custom

Reply:
Hi @mmacy, thanks for your comment. Adding another bullet is good. As for HTML element removal, I will create another Pull Request either for the documents you suggested.